src/Integrations/Insightly/InsightlyClient.php

```diff
@@ -45,179 +48,38 @@ public function __construct(ClientInterface $httpClient, string $apiKey, Pipelin
         $this->pipelineStages = $pipelineStages;
     }
 
-    public function createContact(Contact $contact): Id
+    public function contactResource(): ContactResource
```

Comment:
Nit: `public function contacts()` might be good enough and even clearer maybe, because it's the same as the path on the API then (`/Contacts`)

Reply:
Yes, good suggestion. I will fix this in the follow-up pull request to link a `Contact` to a `Project` or `Opportunity`.